flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  description = "Disruptor";
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    flake-compat = {
+      url = github:edolstra/flake-compat;
+      flake = false;
+    };
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-21.11";
+    nixpkgs_latest.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs_latest, nixpkgs, flake-utils, ... }:
+  flake-utils.lib.eachDefaultSystem (system:
+  let
+    pkgs = nixpkgs.legacyPackages.${system};
+    pkgs_latest = nixpkgs.legacyPackages.${system};
+    dev_shell = pkgs.stdenv.mkDerivation {
+      name = "dev-shell";
+      buildInputs = with pkgs; [
+        cacert
+        coreutils-full
+        curlFull
+        direnv
+        gnutar
+        # Nix 2.5 (as the one from the installator)
```

Comment:
Why 2.5? 
Only as a means of introducing people to new syntax?

Reply:
From the practical standpoint: https://github.com/NixOS/nix/pull/3458
It makes user-specific nix config way more sane in our usecase.